src/Http/Controllers/AjaxMultiselectController.php

```diff
@@ -20,7 +20,8 @@ public function getOptionsByQuery(NovaRequest $request)
 
         $builder->where($label, 'like', "%$query%");
         if ($custom_query) {
-            $builder->whereRaw($custom_query);
+            $custom_query = explode('=', $custom_query);
```

Comment:
Maybe worth using [https://www.php.net/manual/en/function.parse-str.php](https://www.php.net/manual/en/function.parse-str.php) which is designed for parsing query strings? Just a thought. Not a lot wrong with what we've done here though.

```
parse_str($custom_query, $query);
foreach ($query as $key => $val) {
    $builder->whereRaw($key.'= ?', $val);
}
```

Reply:
I like it mate! Just switched it.

Will get it PRd into their repo once approved.